### DISCUSSION TITLE
Few things before i consider this tool for my project

### DISCUSSION THREAD

**Discussion by roker15** [react-page/react-page#1077] [Q&A]
Few things before i consider this tool for my project-- 
1. It says it is powered by "Slate", slate itself is not stable and is in it's beta , how does this tool is stable? 
2. In which form data is saved , can it be saved in html or json?
 3. would i be able to display data saved by this editor in some other editor like "quill"?

**Comment by macrozone** [comment]
@roker15 i suggest to consult the [docs](https://react-page.github.io/docs/)

1. ReactPage is not powered by slate, just the rich text editor. Slate has a 0.x version, which means they can break between minor bumps. It does not mean its not "stable", it means, that you have to be careful when upgrading. We manage the integration of slate, so you don't have to worry about this. We even have a migration system that makes sure that your content will work also in future versions of react-page and slate
2. json
3. no, quill is just a rich-text editor and can't display complex layouts and custom components. Also its json format is different than ours.

**Reply by roker15** [reply]
Still I have some query please answer as quickly as possible becuase i have to decide my editor tool today itself.
1. I want to save data as html. as u said json format can vary so i need a format so that i can render my data anywhere i want.. do u provide any official parser so that json saved by ur editor can be use to render anywhere be it mobile or any web page.
2. i don't see any table support.

**Reply by macrozone** [reply]
1. react-page also renders content, so you use it for editing and displaying. Be aware that react-page does much more than just a html editor, it can also render complex components. This is not possible when you store the data as html. If you need a html editor, you should select a html editor. 
2. indeed that is missing, no one ever required that ;-)

**Reply by roker15** [reply]
> 1. react-page also renders content, so you use it for editing and displaying. Be aware that react-page does much more than just a html editor, it can also render complex components. This is not possible when you store the data as html. If you need a html editor, you should select a html editor.
> 2. indeed that is missing, no one ever required that ;-)

So ---
1. How i can render page on Android if data is in json, i want my blog created in react page to be on my andorid app also?
2. I see no options for changing font size or font style. how to do this?
3. Please provide support for table.... i think it's already implemented in slate... make it complete..
